impl/client/src/test/javascript/pentaho/data/filter/isGreater.spec.js

```diff
@@ -0,0 +1,118 @@
+/*!
+ * Copyright 2017 Pentaho Corporation.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+define([
+  "pentaho/data/filter/isGreater",
+  "pentaho/type/Context",
+  "pentaho/type/complex",
+  "./propertyUtils"
+], function(isGreaterFactory, Context, complexFactory, propertyUtils) {
+
+  "use strict";
+
+  /* global describe:true, it:true, expect:true, beforeEach:true*/
```

Comment:
I suspect that Nuno just continued the practice in other files, so I'll try to justify. This helps IDEA to know that these variables are valid globals. It would be more correct to say:
```js
/* global describe:false, it:false, expect:false, beforeEach:false*/
```
Meaning that the variables can only be read from.

Also, in these days, I prefer the shorter form:
```js
/* globals describe, it, expect, beforeEach*/
```

Anyway, probably there is some editor setting that would achieve the same globally. Is that why you ask?

If this setting could be globally set in an `eslint.rc` file or similar, that would then take effect with proper editor configuration, it would be a better alternative.
Pure editor solutions, on the other hand, are not as good, as these are not stored in the source control system.